packages/babel-plugin-object-enumerables/src/index.js

```diff
@@ -0,0 +1,19 @@
+const keysSource = "((o) => { var r = []; for (var k in o) { r.push(k) } return r; })";
```

Comment:
I suspect there is a better way to do this, most likely by building the AST instead of using a string.